forge-gui/res/cardsfolder/f/freyalises_winds.txt

```diff
@@ -3,7 +3,7 @@ ManaCost:2 G G
 Types:Enchantment
 T:Mode$ Taps | TriggerZones$ Battlefield | Execute$ TrigPutCounter | TriggerDescription$ Whenever a permanent becomes tapped, put a wind counter on it.
 SVar:TrigPutCounter:DB$ PutCounter | Defined$ TriggeredCardLKICopy | CounterType$ WIND | CounterNum$ 1
-R:Event$ Untap | ActiveZones$ Battlefield | ValidCard$ Permanent.counters_GE1_WIND | ReplaceWith$ RepRemoveCounter | UntapStep$ True | Description$ If a permanent with a wind counter on it would untap during its controller's untap step, remove all wind counters from it instead.
+R:Event$ Untap | ActiveZones$ Battlefield | ValidCard$ Permanent.counters_GE1_WIND | ReplaceWith$ RepRemoveCounter | UntapStep$ Player | Description$ If a permanent with a wind counter on it would untap during its controller's untap step, remove all wind counters from it instead.
```

Comment:
this logic seems wrong because you're now only checking that it's any untap step (instead of controller)

Reply:
could add `+ActivePlayerCtrl`?

Reply:
i added ActivePlayerCtrl